docker-compose.yml

```diff
@@ -35,68 +27,74 @@ services:
   mq:
     image: "redis"
     restart: always
-    networks:
-      - fn-network
-    ports:
-      - "6379:6379"
-  fnserver:
+  api:
     restart: always
     depends_on:
       - mq
       - db
       - logstore
-    build: .
-    networks:
-      - fn-network
+    build:
```

Comment:
Network attribute was necessary to let functions be within the Fn network, and the idea was to have a flat network where each component has a binding.